abcclassroom/__main__.py

```diff
@@ -5,6 +5,7 @@
 from . import template
 from . import feedback as fdback
 from . import github
+from . import git
```

Comment:
I think what we want to do here is also import auth in addition to git and github